fixtures.ttar

```diff
@@ -1896,6 +1896,82 @@ Lines: 52
    8       2 sdc2 13126 71 561749 16802 2830 1589 176404 40620 0 10931 50449 0 0 0 0 0 0
 Mode: 664
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Directory: fixtures/proc/flashcache
+Mode: 755
+# ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Directory: fixtures/proc/flashcache/CG0
+Mode: 755
+# ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Path: fixtures/proc/flashcache/CG0/flashcache_errors
+Lines: 10
+disk_read_errors=0 disk_write_errors=0 ssd_read_errors=0 ssd_write_errors=0 memory_alloc_errors=0
+thin_alloc_errors=0
+ssd_errors=0
+verify_errors1=0
+verify_errors2=0
+verify_errors3=0
+verify_errors4=0
+verify_errors5=0
+verify_errors6=0
+verify_errors7=0
+Mode: 644
+# ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Path: fixtures/proc/flashcache/CG0/flashcache_iosize_hist
+Lines: 1
+1:15 2:0 4:52419764 8:4301895 16:601787 32:77257 64:35248 128:29153 256:10846 512:260737 1024:128460 
+Mode: 644
```

Comment:
@SuperQ Could you please tell me what is the `iosize_hist` about and what is the a and b in this `a:b` format?

Reply:
I have no idea, this is just what I found on the system. It doesn't appear to be documented. :-(

Looking over the source, it appears to be a [in size of sectors](https://github.com/facebookarchive/flashcache/blob/437afbfe233e94589948b76743c6489080cdd100/src/flashcache_main.c#L2087-L2093), or 512 bytes.

So we'll have to convert it from non-cumulative sectors, to cumulative bytes.